src/databricks/labs/ucx/config.py

```diff
@@ -74,6 +74,9 @@ class WorkspaceConfig:  # pylint: disable=too-many-instance-attributes
     # [INTERNAL ONLY] Whether the assessment should lint only specific dashboards.
     include_dashboard_ids: list[str] | None = None
 
+    # [INTERNAL ONLY] Whether managed table should be created as external UC table.
+    managed_to_external: bool = True
```

Comment:
```suggestion
    # Whether managed table should be created as external UC table.
    convert_managed_tables_on_external_storage_to_external_tables: bool = False
```

but the main question is: should it be a binary choice or we present 3 options to the customer?